docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+# XMR to BTC Atomic Swap - Automated Swap Backend (ASB)
+
+## Quick Start ASB
+
+- Ensure that you have the Monero Wallet RPC running with `wallet-dir` and `disable-rpc-login`:
+  - `monero-wallet-rpc --stagenet --daemon-host STAGENET-NODE-URL --rpc-bind-port STAGENET-NODE-PORT --disable-rpc-login --wallet-dir PATH/TO/WALLET/DIR`
+- Run the ASB in terminal: `./asb start`
+- Follow the setup wizard in the terminal
+
+Public stagenet nodes:
+
+- `monero-stagenet.exan.tech`, `38081`
+- `stagenet.community.xmr.to`, `38081`
+
+Run `./asb --help` for more information.
+
+## ASB Details
+
+The ASB is a long running daemon that acts as the trading partner to the swap CLI.
+The CLI user is buying XMR (i.e. receives XMR, sends BTC), the ASB service provider is selling XMR (i.e. sends XMR, receives BTC).
+The ASB can handle multiple swaps with different peers concurrently.
+
+The ASB reacts to:
+
+- Quote requests from a CLI
+- Swap execution setup with a CLI
+- Swap execution messages with a CLI
```

Comment:
Not sure this is important in that level of detail. Maybe something like:

```suggestion
The ASB communicates with the CLI on various libp2p-based network protocols.
```

Is good enough without revealing implementation details?